modules/nw-bgp-about.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * networking/bgp_routing/about-bgp-routing.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="nw-bgp-about_{context}"]
+= About Border Gateway Protocol (BGP) routing
+
+{product-title} supports BGP routing through FRRouting (FRR), a free, open source internet routing protocol suite for Linux and UNIX platforms. FRR-K8s is a Kubernetes based daemon set that exposes a subset of the FRR API in a Kubernetes-compliant manner. As a cluster administrator, you can use the FRRConfiguration custom resource (CR) to access FRR services.
+
+[id="supported-platforms_{context}"]
+== Supported platforms
+
+BGP routing is supported on the following infrastructure types:
+
+- Bare metal
+- {vmw-full} on-premise
+
+[id="prerequisites_{context}"]
+== Prerequisites
+
+Use of BGP routing requires that you have properly configured BGP for your network provider. Outages or misconfigurations of your network provider might cause disruptions to your cluster network.
+
+[id="considerations-for-use-with-the-metallb-operator_{context}"]
+== Considerations for use with the MetalLB Operator
+
+The MetalLB Operator is installed as an add-on to the cluster. Deployment of the MetalLB Operator automatically enables FRR-K8s as an additional routing capability provider and uses the FRR-K8s daemon installed by this feature.
+
+Before upgrading to 4.18, any existing FRRConfiguration in the `metallb-system` namespace not managed by the MetalLB operator (added by a cluster administrator or any other component) needs to be copied to the `openshift-frr-k8s` namespace manually, creating the namespace if necessary.
+
+[IMPORTANT]
+====
+If you are using the MetalLB Operator and there are existing `FRRConfiguration` CRs in the `metallb-system` namespace created by cluster administrators or third-party cluster components other than MetalLB Operator, you must ensure that they are copied to the `openshift-frr-k8s` namespace or that those third-party cluster components use the new namespace.
```

Comment:
exiting **custom** FRRConfiguration CR as there will be auto generated FRR configurations.

Reply:
@asood-rh I guess existing FRRConfiguration CR should be moved to `openshift-frr-k8s` namespace to make them re-generated

Reply:
@zhaozhanqi That is correct non metallb configuration called custom configuration need to be manually copied over to openshift-frr-k8s. Was trying to understand what third party components use FRR Configuration other than metallb.